.eslintrc.json

```diff
@@ -7,7 +7,15 @@
     ],
     "parserOptions": {
         "ecmaVersion": 6,
-        "sourceType": "module"
+        "sourceType": "module",
+        "project": [
+            "./tsconfig.json",
+            "./tsconfig-eslint.json",
+            "./packages/generator-langium/tsconfig.json",
+            "./packages/langium-cli/tsconfig.json",
+            "./packages/langium-sprotty/tsconfig.json",
+            "./packages/langium-vscode/tsconfig.json"
```

Comment:
Are these necessary? It didn't seem to make a difference when I tried removing these lines.

Reply:
I just noticed, that after restarting ESlint server linting in editor fails :( 
We will probably need to try something else. It seem to behave differently wenn running eslint in CLI and using it in the editor